live/src/main/java/org/dbpedia/extraction/live/storage/JSONCache.scala

```diff
@@ -119,8 +122,9 @@ class JSONCache(pageID: Long, pageTitle: String) {
       cacheExists = true
       if (cacheObj.json.equals("")) return
 
-      val json: Option[Any] = JSON.parseFull(cacheObj.json)
-      val map: Map[String, Any] = json.get.asInstanceOf[Map[String, Any]]
+      val mapper = new ObjectMapper() with ScalaObjectMapper
+      mapper.registerModule(DefaultScalaModule)
```

Comment:
Not very familiar with this library but should we call `registerModule` on every instantiation?